src/Inspection.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+namespace RicorocksDigitalAgency\Morpher;
+
+
+class Inspection
+{
+    public $beforeHook;
+    public $afterHook;
+
+    public function before(callable $closure)
+    {
+        $this->beforeHook = $closure;
```

Comment:
Should we allow for multiple hooks in the same inspection? To allow for such syntax as:

```php
Morph::test(Foo::class)
   ->before(...)
   ->before(...)
   ->after(...)
```

Reply:
Added. Simplifies the logic quite nicely too into a one-liner.